.github/workflows/chromatic.yml

```
@@ -0,0 +1,15 @@
+name: "Chromatic"
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - run: |
+        npm install
+        npm run build:storybook
+    - uses: chromaui/action@v1
+      with: 
+        projectToken: 3evi132fpys
+        token: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
        projectToken: 3evi132fpys
        token: ${{ secrets.GITHUB_TOKEN }}
        storybookBuildDir: docs/dist

```

This way it knows which folder to upload, which was already build be the `build:storybook` script.

https://github.com/chromaui/action#usage-all-options

We should, hope that's possible, also try and get a new `projectToken` and hide it behind a github secret.